test/ebpf/src/endpoint_prog.c

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "bpf_helpers.h"
+#include "endpoint_prog.h"
+#include "ebpf_structs.h"
```

Comment:
```suggestion
```
bpf_helpers.h on windows internally includes ebpf_structs.h so you don't need to do it again (and removing it makes your code more cross-plat windows/linux)